### ISSUE TITLE
HTML report is erroneous on beta environment

### ISSUE THREAD

**Issue by dstenger** [opengeospatial/teamengine#252]
HTML report cannot retrieved by neither REST API nor Web Browser Interface.
REST API returns exception instead of zip file.
Web Browser Interface just returns XML report (HTML report is expected).

Logging in catalina.out.
REST API:
```
Oct 09, 2017 4:46:10 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet [TestSuiteController] in context with path [/te2] threw exception
java.lang.NullPointerException
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.executeTestRun(TestRunResource.java:243)
	at com.occamlab.te.spi.jaxrs.resources.TestRunResource.handleHtmlGet(TestRunResource.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.jersey.spi.container.JavaMethodInvokerFactory$1.invoke(JavaMethodInvokerFactory.java:60)
	at com.sun.jersey.server.impl.model.method.dispatch.AbstractResourceMethodDispatchProvider$ResponseOutInvoker._dispatch(AbstractResourceMethodDispatchProvider.java:205)
```
Web Browser Interface:
```
Warning: at xsl:variable on line 83 of file:/srv/srv/teamengine-te2-201710/webapps/te2/WEB-INF/classes/com/occamlab/te/web/viewlog.xsl:
  SXWN9001: A variable with no following sibling instructions has no effect
index.html (Permission denied)null
index.html (Permission denied)null
Warning: at xsl:variable on line 83 of file:/srv/srv/teamengine-te2-201710/webapps/te2/WEB-INF/classes/com/occamlab/te/web/viewlog.xsl:
  SXWN9001: A variable with no following sibling instructions has no effect
index.html (Permission denied)null
```

**Comment by lgoltz** [comment]
During transformation from EARl to HTML the source html file (index.html) was missing causing a FileNotFound exception. The exception handling and logging was improved and the creation of the index.html invoked. This had to be done in two places, in HtmlReport for the REST API and TECore for the WEB GUI.


Solved by #253

**Comment by dstenger** [comment]
Fix was successfully tested on beta environment.
